pages/common/hsw-cli.md

```diff
@@ -0,0 +1,36 @@
+# hsw-cli
+
+> The command line REST tool for the Handshake wallet.
+> More information: <https://npmjs.com/package/hs-client>.
+
+- Unlock the current wallet:
+
+`hsw-cli unlock {{passphrase}} {{timeout}}`
+
+- Lock the current wallet:
+
+`hsw-cli lock`
+
+- View the current wallets details:
+
+`hsw-cli get`
+
+- View the current wallets balance:
+
+`hsw-cli balance`
+
+- View the current wallets transaction history:
+
+`hsw-cli history`
+
+- Send a transaction to the specified address:
+
+`hsw-cli send {{address}} {{value}}`
```

Comment:
What kind of value is this? Perhaps a concrete example might be better here.

Reply:
It's the amount to send. So, `1.03` for example.

I guess maybe `coin_amount` may be more fitting?

Reply:
I think directly mentioning 1.03 might be simpler. And mention "coin_amount" in the description.